components/eventing-controller/pkg/backend/jetstreamv2/utils.go

```diff
@@ -103,17 +118,34 @@ func getStreamConfig(natsConfig env.NatsConfig) (*nats.StreamConfig, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	retentionPolicy, err := toJetStreamRetentionPolicy(natsConfig.JSStreamRetentionPolicy)
 	if err != nil {
 		return nil, err
 	}
+
+	// Quantities must not be empty. So we default here to "-1"
+	if natsConfig.JSStreamMaxBytes == "" {
+		natsConfig.JSStreamMaxBytes = "-1"
```

Comment:
why are we defaulting, if the `natsConfig.JSStreamMaxBytes` defaulted to "-1" ?
`	JSStreamMaxBytes        string envconfig:"JS_STREAM_MAX_BYTES" default:"-1"`